### ISSUE TITLE
[Bug]: prefer navigating to the model property/method/scope if the doc comment exists

### ISSUE THREAD

**Issue by panstromek** [laravel-idea/plugin#953]
### Bug description

Currently, if there's scope/relation/attribute mentioned in the doc comment in the class (usually generated with `laravel-ide-helper`), `jump-to-definition` jumps to the comment, instead of the actual definition of the thing. If you delete the comment, it jumps correctly.

### Operating system

None

### Steps to reproduce

go to `popular()` call in the snippet below and  invoke 'go to definition'

```php

/**
 * App\User
 * @method static Builder|User popular()
 **/
class User {
    public function scopePopular(Builder $query): void {}
}

// code
$users = User::popular(); // 'go to definition' here will navigate to the comment, not to the scope

```

**Comment by adelf** [comment]
Hello. There is own Helper code generator by Laravel Idea. And this feature already works there.

![telegram-cloud-photo-size-2-5460928575176823912-x](https://github.com/laravel-idea/plugin/assets/2818394/eda831ee-06f9-4e83-922e-0190eb7baf3c)

**Comment by panstromek** [comment]
Yes, I know this, but the problem is when enabling the plugin for existing codebase which has doc comments on every model. Generating helper files in the plugin doesn't help as the ones inline are always prioritized first.

**Comment by panstromek** [comment]
Note that I can't delete the doc comments, because they contain documentation (hand written comments) and also not everybody on the project uses this plugin.

**Comment by panstromek** [comment]
To be clear - I'm not talking about doc comments in generated files, but doc comments written directly in model class files. Maybe this wasn't obvious from the description.

**Comment by adelf** [comment]
Got it. I'll check what can be done.

**Comment by MichelJonkman** [comment]
> Note that I can't delete the doc comments, because they contain documentation (hand written comments) and also not everybody on the project uses this plugin.

Another option would be to have `laravel-ide-helper` place the helper code in separate files and then developers using the plugin can exclude those files and use the built in helper generator.
